CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Rabbitmq username and password [#61](https://github.com/ncsa/standalone-smm-analytics/issues/90)
+- Email notification subject space issue [#102](https://github.com/ncsa/standalone-smm-analytics/issues/102)
```

Comment:
Is it ok to put this enry in the changelog under each folder. I was mostly using the project-wise changelog for project wide issue. Hence they don't have any release version either.

Reply:
sure I will update it

Reply:
I moved it to each changelogs in the folder